src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/envelopehandlers/SupplementaryEvidenceWithOcrHandler.java

```diff
@@ -15,6 +15,8 @@
 @Service
 public class SupplementaryEvidenceWithOcrHandler {
 
+    public static final int MAX_RETRIES = 2;
```

Comment:
can we make this value configurable?

Reply:
do we want to have it different depending on which environment it's running?

Reply:
no, just thought it would be easier if we want to increase the retries. it's up to you

Reply:
how do we retry ? if it fails do we try just after it?